projects/plugins/jetpack/modules/comments/comments.php

```diff
@@ -311,7 +311,12 @@ public function comment_form_after() {
 			$params['has_cookie_consent']  = (int) ! empty( $commenter['comment_author_email'] );
 		}
 
-		$blog_token        = ( new Tokens() )->get_access_token();
+		$blog_token = ( new Tokens() )->get_access_token();
+		// If we have no token, bail.
+		if ( ! $blog_token || is_wp_error( $blog_token ) ) {
+			return;
```

Comment:
Should we output some sort of indication that there was an error, instead of having no comment form being output with no explanation?

Reply:
That's a good point. What do you think about extracting this a bit, so we can bail entirely and get the default core comment form when we do not have a healthy connection to WordPress.com?
e9f60bc7a83f15736f295ce4ada4047dc212859d

Reply:
Fine with me.